internal/generator/backend/template/zkpschemes/groth16/mpcsetup/lagrange.go.tmpl

```diff
@@ -0,0 +1,201 @@
+import (
+	"math/big"
+	"math/bits"
+	"runtime"
+
+	"github.com/consensys/gnark-crypto/ecc"
+
+	{{- template "import_fr" . }}
+	{{- template "import_curve" . }}
+	{{- template "import_fft" . }}
+	"github.com/consensys/gnark/internal/utils"
+)
+
+
+
+func LagrangeCoeffsG1(powers []curve.G1Affine, size int) []curve.G1Affine {
```

Comment:
I think it can be made private. Used only within package and imo makes looking at the package methods otherwise difficult.